carbonmark-api/src/utils/helpers/utils.constants.ts

```diff
@@ -1,9 +1,9 @@
 import { TOKEN_ADDRESSES } from "../../app.constants";
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-assertions -- unable to type environment variables
-const ENV = (process.env.VERCEL_ENV ?? "development") as
-  | "development"
-  | "production";
+const ENV = (
+  process.env.VERCEL_ENV !== "production" ? "development" : "production"
+) as "development" | "production";
```

Comment:
I do not know how it worked before for preview environments

Reply:
Did this fix and issue you were having? the original seems fine to me (VERCEL_ENV is only undefined when not on VERCEL (so it has a value in preview environments)

Reply:
@0xemc  Yeah, it was refusing to deploy on preview environmenets because `process.env.VERCEL_ENV` was equal to `preview`

Reply:
When you say "refusing the deploy" do you mean the postman tests were failing? Because they should be on this preview build (we're trying to create a release that points to a broken subgraph)